milatools/cli/commands.py

```diff
@@ -27,6 +27,8 @@
 from invoke import UnexpectedExit
 from typing_extensions import TypedDict
 
+from milatools.cli.code_command import set_remote_ssh_vscode_settings
```

Comment:
The other imports are relative, for consistency this one should be as well.

Reply:
Fixed in fc61b941fdd5b5ed907b1f667c24a5dd1cd20c10